content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
Which is (ironically) exactly the reason why I want to link it here. I talked to @tomcz about the necessary migration work for Hystrix, and then he told me he needs to convert the Clojure wrapper, too. It was that lack of visibility that made us not see it.

If you think a wrapper shouldn't be linked here, perhaps it would be better located where we also put gokit and such?